librz/bin/dwarf/op.inc

```diff
@@ -846,6 +846,81 @@ bool compute_pc(RzBuffer *pc, const RzBuffer *bytecode, st16 offset) {
 	return rz_buf_seek(pc, offset, RZ_BUF_CUR) >= 0;
 }
 
+ValueType ValueType_from_name(const char *name) {
+	if (strcmp(name, "int") == 0) {
+		return I32;
+	}
+	if (strcmp(name, "unsigned") == 0) {
+		return U32;
+	}
+	if (strcmp(name, "size_t") == 0) {
+		return U64;
```

Comment:
This is incorrect since target-dependent, same for `unsigned` and `int`

Reply:
You can see how it's done (and also use) by using the API from RzType: `rz_type_db_base_get_bitsize()` - it calculates the bitsize of the type, depending on what has been loaded into the RzTypeDB.

Reply:
RzType is already a dependency of RzBin, so could be used without any problem

Reply:
It requires RzTypeDB to be loaded though, through `rz_type_db_init()` function